origin.spec

```diff
@@ -289,6 +289,8 @@ done
 install -d -m 755 %{buildroot}%{_sysconfdir}/systemd/system.conf.d/
 install -p -m 644 contrib/systemd/origin-accounting.conf %{buildroot}%{_sysconfdir}/systemd/system.conf.d/
 
+%clean
```

Comment:
Is this necessary given the --noclean flag and doesn't this mean that cleaning is effectively disabled entirely regardless of rpmbuild options?